### ISSUE TITLE
Make the persistent process take the .packages file next to it if none has been provided

### ISSUE THREAD

**Issue by ricowind** [dart-lang/fletch#145]
This will allow for easy bundling of a default .packages file in the initial sdk bundle

**Comment by sgjesse** [comment]
What does "next to it" mean? Shouldn't it do this:

"The Dart runtimes, such as the VM, dartjs, and analyzer, will traverse up directories, from the entry point script, to find a .packages. Thus, you do not need a .packages next to each script that you need to run or compile."

(from http://blog.sethladd.com/2015/08/new-dart-sdk-eliminates-symlinks.html).

**Comment by peter-ahe-google** [comment]
Consider this situation:

The user installs the Fletch SDK in /usr/local/fletch-sdk.

The user adds /usr/local/fletch-sdk/bin to their PATH.

The user run:

```console
$ fletch run fun_with_gpio.dart
```

In that case we just want to use the default .packages from the SDK that tells fletchc where to find the gpio package.

This may be a short-term thing.

**Comment by sgjesse** [comment]
So we supply a default .packages for resolving all packages in pkg that will be used no matter where your Dart file is.

Sounds good to me.

General support for .packages is a separate issue then.

**Comment by peter-ahe-google** [comment]
Proposed fix: [CL 1367723002](https://codereview.chromium.org/1367723002/)

With this, the persistent process will look for a file named `fletch-sdk.packages` next to the Dart VM running the persistent process.

I would have liked to make it relative to `driver_main.dart`, but that is blocked on [Dart issue 23951](https://github.com/dart-lang/sdk/issues/23951).
